ansible/roles/datadog/tasks/main.yml

```diff
@@ -38,5 +38,6 @@
     accept_hostkey: yes
     update: yes
   sudo_user: "{{ cchq_user }}"
+  notify: restart datadog
```

Comment:
It should only do it if there are new changes that get pulled (though the docs don't mention anything about it examining the source confirms). Also not a huge issue to restart the datadog agent.